src/EFCore.MySql/Utilities/StringExt.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Pomelo Foundation. All rights reserved.
+// Licensed under the MIT. See LICENSE in the project root for license information.
+
+// ReSharper disable once CheckNamespace
+namespace System.Text
+{
+    public static class StringExt
+    {
+        public static bool Match(this string match, string against)
+        {
+            throw new NotImplementedException("Should not be called from client code.");
+        }
+
+        public static bool MatchInBooleanMode(this string match, string against)
+        {
+            throw new NotImplementedException("Should not be called from client code.");
+        }
+
+        public static bool MatchWithQueryExpansion(this string match, string against)
+        {
+            throw new NotImplementedException("Should not be called from client code.");
+        }
+    }
+}
```

Comment:
I am not convinced that we should add a `Match()` string extension method. I think we should handle this similar to `LIKE`, that just has a `DbFunctions` extension method. So I think this should be removed.

If we decide to include this anyway, this should be renamed to something more expressive (e.g. `MySqlMatchStringExtensions`).